contracts/utils/Swap.sol

```diff
@@ -118,9 +118,11 @@ contract Swap is ISwap, ReentrancyGuard, Ownable {
             biddingToken != address(0),
             "BiddingToken cannot be the zero address"
         );
+        require(totalSize > 0, "TotalSize must be larger than zero");
         require(minPrice > 0, "MinPrice must be larger than zero");
         require(minBidSize > 0, "MinBidSize must be larger than zero");
```

Comment:
If `minBidSize > 0` then `totalSize >0` is implied, since we already did `require(minBidSize <= totalSize)`.

We can remove `require(minBidSize > 0)`